src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -252,6 +252,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'exclusiveMinimum': 0,
 				'markdownDescription': localize('limitEditorsMaximum', "Controls the maximum number of opened editors. Use the `#workbench.editor.limit.perEditorGroup#` setting to control this limit per editor group or across all groups.")
 			},
+			'workbench.editor.limit.ignoreDirtyTabs': {
+				'type': 'boolean',
+				'default': false,
+				'description': localize('limitEditorsIgoreDirtyTabs', "Controls if the maximum number of opened editors should ignore dirty tabs. When enabled, ignore the dirty tabs")
```

Comment:
Suggest: `Controls if the maximum number of opened editors should exclude dirty editors for the configured limit.`